templates/patterns/list_with_illustration/pattern-list-with-illustration.html.twig

```diff
@@ -0,0 +1,42 @@
+{#
+/**
+ * @file
+ * List with illustration.
+ */
+#}
+
+{# Set the columns to 1 if vertical variants. #}
+{% if 'vertical' in variant %}
+  {% set column = 1 %}
+{% endif %}
+
+{# Set the zebra to false if horizontal variants. #}
+{% if 'horizontal' in variant %}
+  {% set zebra = false %}
+{% endif %}
+
+{% set _items = [] %}
+{% for item in items %}
+  {% if 'icon' in variant %}
+    {# Remove image information if an icon variant. #}
+    {% set _item = item|merge({'image': {}}) %}
+    {# Transform icon into ecl array. #}
+    {% if _item.icon %}
+      {% set _item = _item|merge({'icon': to_ecl_icon(item.icon)}) %}
```

Comment:
Unfortunately it won't work with custom icons, like flags.

Reply:
Looks like ECL icons have to be used in any case since they are used in components/twig-component-list-illustration/ecl-list-illustration-item.html.twig.

We also need to use flag icons, so need ability to set custom path for icons. I've achieved it by preparing icon in the template, not in the pattern. @imanoleguskiza please have a look commit https://github.com/openeuropa/oe_theme/pull/1006/commits/2ca33c65aa11bedf671cfe00d2fe6c61fb2b2c2d.
I also used "landscape" instead of "large" for variant name to avoid additional mapping between paragraph values and pattern. Please check how I use it in https://github.com/openeuropa/oe_theme/pull/1009/files. Are my changes acceptable?